crates/proto/Cargo.toml

```diff
@@ -84,7 +84,7 @@ js-sys = { workspace = true, optional = true }
 lazy_static.workspace = true
 native-tls = { workspace = true, optional = true }
 openssl = { workspace = true, features = ["v102", "v110"], optional = true }
-quinn = { workspace = true, optional = true }
+quinn = { workspace = true, optional = true, features = ["runtime-tokio", "tls-rustls"] }
```

Comment:
`log` was enabled by `default-features` as well, should I add that as well?

Reply:
I have no opinion on this...

Reply:
I would rather add it if we are unsure about this, at least it will keep the current behavior in place.

Let me know if you want me to do anything here.

Reply:
Okay, let's do it.

Reply:
Done.